tests.sh

```diff
@@ -165,6 +165,14 @@ build_csharp() {
 
   # Run csharp compatibility test between last released and the current version.
   csharp/compatibility_tests/v3.0.0/test.sh $LAST_RELEASED
+  
+  # Regression test for https://github.com/protocolbuffers/protobuf/issues/9526
+  # - all line endings in .proto and .cs (and .csproj) files should be LF.
+  if git ls-files --eol csharp | grep -E '\.cs|\.proto' | grep -v w/lf
```

Comment:
I didn't, but I think we're okay... it will fail if we no longer have any C# or proto files, but I think that's a case which is somewhat worse than just line-endings being wrong anyway :)